jbmc/regression/jbmc-concurrency/synchronized-blocks-throw/Sync.java

```diff
@@ -0,0 +1,35 @@
+import org.cprover.CProver;
+
+public class Sync
+{
+  int field;
+  public static void main(String[] args)
+  {
+    Sync o = new Sync();
+    try
+    {
+      o.field=0;
+      synchronized (o)
+      {
+        // Make sure the monitor is taken.
+        assert(CProver.getMonitorCount(o) == 1);
```

Comment:
This will not work because the body of`CProver.getMonitorCount()` is not part of the models library...

Reply:
Yes, nonetheless I thought it was an important  test that should be included for now, when we separate them we can either disable it or preferably  better add a getter method  in the 'java.lang.Object' model instead.